frontend/src/component/Activity/ActivityList.jsx

```diff
@@ -49,6 +49,7 @@ const ENTITY_TYPE_COMPONENT_CONSTRUCTOR = new Map([
       : (
         <FeedItemWithPreview
           breadcrumbsList={breadcrumbsList}
+          success={activity.success}
```

Comment:
success seems a bit ambiguous, could you rename to something more explicit?

Reply:
I am going to rename this to available indeed.

Reply:
while I agree for `activity.available` I think this it should be named differently for FeedItemWithPreview props because its behavior is pretty specific

Reply:
mmh actually it might not be that important

Reply:
I applied available everywhere. I think it is clean but I am willing to change. Tell me if you have an idea.

Reply:
or maybe it is ><
I mean, the available props is directly linked to the content

Reply:
you are right, I'll rename

Reply:
done